groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group9/CobaltChain.groovy

```diff
@@ -60,7 +60,7 @@ ELECTROLYTIC_CELL.recipeBuilder()
         .notConsumable(metaitem('stickCobalt'))
         .notConsumable(metaitem('graphite_electrode'))
         .fluidInputs(fluid('cobalt_sulfate_solution') * 1000)
-        .outputs(metaitem('dustCobalt'))
+        .outputs(metaitem('dustHighPurityCobaltCobalt'))
```

Comment:
What? This should be `dustHighPurityCobalt`, not `dustHighPurityCobaltCobalt`!

Also, you really need to ask if this is the correct solution, given how much we've been changing this.